docs/source/alfacase_definitions/CaseOutputDescription.txt

```diff
@@ -7,28 +7,28 @@
         class CaseOutputDescription
             automatic_trend_frequency: bool = True
             automatic_trend_frequency: bool = True
-            trends: \ :class:`List <typing.List>`\[\ :class:`TrendOutputDescription <TrendOutputDescription>`\] = []
-            trend_frequency: \ :class:`Scalar <barril.units.Scalar>`\  = Scalar(0.1, 's', 'time')
+            trends: \ :class:`List <typing.List>`[\ :class:`TrendOutputDescription <TrendOutputDescription>`] = []
```

Comment:
Nops the escape character is needed, see:
![image](https://user-images.githubusercontent.com/5083518/114933604-383aae80-9e0f-11eb-954d-0cc84fefdd97.png)